src/legend.ts

```diff
@@ -0,0 +1,239 @@
+export module LegendHTML {
+    function genSVGArrow(color: string, dashed: boolean, double: boolean, start: string, end: string, urlName: string, markerID: string, doubleMarkerID: string): SVGElement{
+
+        let line = document.createElementNS(svgURL, "line")
+        line.setAttribute("x1", "0")
+        if (double) {
+            line.setAttribute("x1", "100")
+        }
+        line.setAttribute("y1", "50")
+        line.setAttribute("x2", "250")
+        line.setAttribute("y2", "50")
+        line.setAttribute("stroke", color)
+        line.setAttribute("stroke-width", "8")
+        line.setAttribute("marker-end", urlName)
+
+         if (dashed) {
```

Comment:
Can you follow the spacing conventions of the rest of the code, and put any `else` or `else if` statements on the same line as the closing curly brace? i.e.,

```
if (dashed) {
    ...
} else if (double) {
    ...
}
```